### ISSUE TITLE
get a haircut

### ISSUE THREAD

**Issue by iangilman** [iangilman/me#4]
you're starting to look a little shaggy there…

**Comment by eldang** [comment]
After how many days does the label switch from enhancement to bug?

**Comment by iangilman** [comment]
Good point! We may be heading into that territory already…

Truth be told, I'm concerned that even once I close this issue it'll just pop up later as a regression. I wonder what sort of unit tests we can put in place to keep it from recurring?

**Comment by eldang** [comment]
There was a teacher at my old school who would measure boys' hair with a ruler to see if it complied with the uniform rules.  Not that I'm suggesting you emulate that....

**Comment by iangilman** [comment]
Yeah, I guess it's not a test I need; I'm well enough aware when the issue resurfaces.

I suppose this is a classic example of the difference between problems you can solve versus problems you have to manage.

**Comment by iangilman** [comment]
The fix for this issue has landed! Verified resolved.

**Comment by iangilman** [comment]
Sure enough, we have a regression! At least the unit test caught it… Now I just need to find someone willing to submit a patch.
